eva-accession-import-dbsnp2/src/main/java/uk/ac/ebi/eva/accession/dbsnp2/model/DbsnpJson.java

```diff
@@ -0,0 +1,126 @@
+/*
+ * Copyright 2019 EMBL - European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package uk.ac.ebi.eva.accession.dbsnp2.model;
+
+import org.springframework.batch.core.configuration.annotation.StepScope;
+import uk.ac.ebi.eva.accession.core.ISubmittedVariant;
+
+import java.time.LocalDateTime;
+
+/**
+ * Core fields of a variant extracted from a dbSNP JSON file.
+ */
+@StepScope
```

Comment:
This is not necessary, the scope applies when declaring something to be used as a `@Bean`. This class lifecycle is not managed by Spring.